src/Uno.UI/UI/Xaml/Controls/TimePicker/UnoTimePickerDialog.Android.cs

```diff
@@ -74,6 +76,8 @@ private void SetMinuteIncrement()
 
 			if (minutePicker != null)
 			{
+				IsInSpinnerMode = true;
```

Comment:
it's a little weird that IsInSpinnerMode is set as a side-effect of calling SetMinuteIncrement.

Reply:
Its ok for now I suppose, because it doesn't yet support changing the MinuteIncrement on TimePickerDialog, if it changes on the TimePicker, it would create a new dialog anyway on next "open", correct?

Reply:
yes it could be refactored a little better.. lets keep it like this for now.. i'll refactor and test another way after